Makefile

```diff
@@ -37,7 +37,7 @@ test: | tests/cli.toml
 	pytest -v tests
 
 servedocs:
-	mkdocs serve -w docs_theme -w pulp-glue/pulp_glue -w pulpcore/cli/common/generic.py
+	pulp-docs serve
```

Comment:
Nice!
But the -w options stopped working. As well as watching the "usual" docs files.

Reply:
Well this is the beauty of Make we can just add all the files relevant to this repository here once we can pass -w to pulp-docs. No need to have auto on all the other repositories at that time.
But I was really wondering, because watch did not even work for the _normal_ docs files.

Reply:
https://github.com/pulp/pulp-docs/issues/9